MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -89,6 +91,38 @@ public class AudioFileProber
                 Fetch(item, result, cancellationToken);
             }
 
+            var libraryOptions = _libraryManager.GetLibraryOptions(item);
+
+            if (libraryOptions.EnableLUFSScan)
+            {
+                string output;
+                using (System.Diagnostics.Process proc = new System.Diagnostics.Process())
```

Comment:
Very minor thing, but you should follow the "coding style standards" for how we use Process. You can search around to find it.

Reply:
Do you mean adding "using System.Diagnostics" to the top of the file and then changing line 99 so it reads "using (Process proc = new Process())"? if so i can implement that now.

Reply:
No, I mean this https://github.com/jellyfin/jellyfin/blob/910617bbc3b960ff6a1e1c18c27a0b42a4be9ee0/MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs#L398-L406. How we initialize it and the extra options.

Reply:
Updated now please advise if it is not to code style standards. Thanks! :)